### PR TITLE
feat: toggle button has implemented

### PR DESCRIPTION
**Pull request by AlAminKh03**
- toggle button has implemented to switch active to all contacts

### CODE REVIEW

File: src/components/GuestDetailsInfo.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const GuestDetailsInfo = ({
         variables: {
           createContactInput: {
             ...values,
-            idNo: Number(values.idNo),
+            idNo: Number(values.idNo).toString(),
```
Comment (ahmad-ml) [inline src/components/GuestDetailsInfo.tsx:76]: it is already sting ... Could you check again?

**Review by ahmad-ml — CHANGES_REQUESTED** [review]
Could you sort the guest list as per name A-Z ?

**Review by ahmad-ml — CHANGES_REQUESTED** [review]
Could you sort the guest list as per name A-Z ?

File: src/components/GuestDetailsInfo.tsx (hunk 1 above)
Comment (AlAminKh03) [reply src/components/GuestDetailsInfo.tsx:76]: Fixed, sir.

**Comment by AlAminKh03** [comment]
just implemented the sorting of contacts, waiting for your check sir.

**Pull request closed without merging**